logstash-core/benchmarks/src/main/java/org/logstash/benchmark/QueueBenchmark.java

```diff
@@ -87,8 +87,8 @@ private static Settings settings() {
             .capacity(256 * 1024 * 1024)
             .queueMaxBytes(Long.MAX_VALUE)
             .elementIOFactory(MmapPageIO::new)
-            .checkpointMaxWrites(50_000)
-            .checkpointMaxAcks(50_000)
+            .checkpointMaxWrites(1024)
```

Comment:
It's not clear why this change was made. Can you omit it or include the reason in the commit message?